classes/Domain/Entity/Mapper/Talk.php

```diff
@@ -23,10 +23,10 @@ class Talk extends Mapper
      * user has favourited this talk
      *
      * @param integer $admin_user_id
-     * @param string $order_by
-     * @param string $sort Sort Direction
-     * @throws InvalidArgumentException If order by is not in white list
-     * @return array
+     * @param array $options
+     * @return array If order by is not in white list
+     * @internal param string $order_by
```

Comment:
Thanks @jsundquist. I love when PhpStorm does this (`@internal` annotations on missing params). I blame @GeeH.